R/api.R

```diff
@@ -30,3 +30,41 @@ root <- function() {
   )
   lapply(versions, function(v) scalar(as.character(v)))
 }
+
+##' @porcelain GET /metadata => json(metadata)
+metadata <- function() {
+  # JIDEA-62: we will use relevant model version - from qs if specified, else
+  # from latest available metadata file
+  model_version <- scalar("0.1.0")
```

Comment:
Could this pull from the package version for now? The model structure and data versions are all bundled together in the package version at the moment anyway.